frontend/ui/lib/widgets/create_trip_widget.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:tripmeout/widgets/map_widget.dart';
+
+//TODO: Add loading screen after to the Create Trip Widget
+//TODO: Get rid of the newInformation text on the page.
+
+class CreateTripWidget extends StatefulWidget {
+  @override
+  _CreateTripWidgetState createState() => _CreateTripWidgetState();
+}
+
+class _CreateTripWidgetState extends State<CreateTripWidget> {
+  String place = "No Input";
+  int radius = 0;
+  String newInformation = 'Grabbed info placed here.';
+  changeText() {
```

Comment:
can you format this? and also declare the return type?
Like `void changeText() {`